lib/phases/reboot.js

```diff
@@ -63,6 +63,7 @@ Reboot.prototype.reboot = function() {
       return phase._restart();
     })
     .then(function(time) {
+      phase.setup();
```

Comment:
Right, I was thinking after the restart but before phase.setup()... but if 'adb log' is not affected by a potential failing or delayed 'adb forward' then there's no point. I guess if ultimately the 'adb forward' fails the entire test is going to stop anyway with device not found.